copulas/univariate/student_t.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+from scipy.stats import t
+
+from copulas.univariate.base import BoundedType, ParametricType, ScipyModel
+
+
+class StudentTUnivariate(ScipyModel):
+    """Wrapper around scipy.stats.t.
+
+    Documentation: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.t.html
+    """
+
+    PARAMETRIC = ParametricType.PARAMETRIC
+    BOUNDED = BoundedType.UNBOUNDED
+
+    MODEL_CLASS = t
+
+    def _fit_constant(self, X):
+        self._params = {
+            'df': 100,
+            'loc': np.unique(X)[0],
+            'scale': 0
+        }
+
+    def _fit(self, X):
+        self._params = {
+            'df': t.fit(X)[0],
```

Comment:
Here you are computing `fit` three times unnecessarily.
I would compute it once and then make the dictionary in a second step:

```python
        df, loc, scale = t.fit(X)
        self._params = {
            'df': df,
            'loc': loc,
            'scale': scale,
        }
```